Loop/Managers/Service.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import LoopKit
+import LoopKitUI
```

Comment:
Is this needed?

Reply:
Yes, the `ServiceDescriptor` type was moved from `LoopKit` to `LoopKitUI`.